contracts/sol6/Dao/mock/MockStakingContract.sol

```diff
@@ -27,11 +27,11 @@ contract MockStakingContract is KyberStaking {
         return stakerPerEpochData[epoch][staker].delegatedStake;
     }
 
-    function getDelegatedAddressValue(address staker, uint256 epoch)
+    function getRepresentativeValue(address staker, uint256 epoch)
```

Comment:
why do we need this getter in mock?

Reply:
Using this to get raw data of representative without looping back, so can test correct data updated for each epoch

Reply:
OK